<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1171</width>
        <height>531</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1169</width>
         <height>529</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Text :)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Text HERE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>550</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Principal">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Generator</string>
    </property>
    <addaction name="actionDice_gen"/>
    <addaction name="actionPNJ_gen"/>
    <addaction name="actionItem_Gen"/>
    <addaction name="actionCity_Gen"/>
    <addaction name="actionTreasur_Gen"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Data Base</string>
    </property>
    <addaction name="actionSpells"/>
    <addaction name="actionItems"/>
    <addaction name="actionGift"/>
    <addaction name="actionMasteries"/>
   </widget>
   <widget class="QMenu" name="menuNextPage">
    <property name="title">
     <string>Page</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
   </widget>
   <addaction name="menuMenu_Principal"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuOutils"/>
   <addaction name="menuNextPage"/>
  </widget>
  <action name="actionDice_gen">
   <property name="text">
    <string>Dice gen</string>
   </property>
  </action>
  <action name="actionSpells">
   <property name="text">
    <string>Spells</string>
   </property>
  </action>
  <action name="actionItems">
   <property name="text">
    <string>Items</string>
   </property>
  </action>
  <action name="actionGift">
   <property name="text">
    <string>Gift</string>
   </property>
  </action>
  <action name="actionMasteries">
   <property name="text">
    <string>Masteries</string>
   </property>
  </action>
  <action name="actionPNJ_gen">
   <property name="text">
    <string>PNJ gen</string>
   </property>
  </action>
  <action name="actionItem_Gen">
   <property name="text">
    <string>Item Gen</string>
   </property>
  </action>
  <action name="actionCity_Gen">
   <property name="text">
    <string>City Gen</string>
   </property>
  </action>
  <action name="actionTreasur_Gen">
   <property name="text">
    <string>Treasure Gen</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
